### DISCUSSION TITLE
Question For How To Obtain Failure Logs

### DISCUSSION THREAD

**Discussion by SamuelBanya** [retronas/discussions#118] [General]
Hey there, how do I obtain related failure logs to share for bugs?

Asking since I have been trying to setup a Debian 11 VM guest in Virtualbox with a Windows 11 host that is passing in its networking via a bridged adapter using the Ethernet adapter with a hooked up Ethernet cable to my router.

Often, I am able to access the samba installation process which just seems to timeout with just green text to reach out for installing the Ansible playbook items. However upon checking the Samba service itself in the related service menu in retronas, it complains that the service was not found or installed.

Would want to share logs to help figure this out thanks.

**Comment by sairuk** [comment]
we will only support the retronas side of this, retronas installer logs available: `/opt/retronas/log/`

linux standard logging either through `journalctl` or log files in `/var/log`
